modules/oadp-deleting-vsbs-after-scheduler.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-deleting-vsbs-after-scheduler_{context}"]
+= Deleting VSBs after a backup procedure that is either run by a scheduler or is run repetitively
+
+If you use a scheduler to run backups involving Data Mover or you run such backups repetitively, you need to perform VSB cleanup after each backup. Otherwise, you might have inconsistent content in your Velero restore operations.
+
+To avoid issues with the Velero scheduler when you use Data Mover in your backup operations, use a Kubernetes `cron` job that includes the backup command and the VSB cleanup.
```

Comment:
It's not necessary to mark up `cron` here. We use "cron job" generically throughout the docs regularly.